modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -25,6 +25,8 @@ class Veteran
     delegate :birls_id, to: :mvi, allow_nil: true
     delegate :participant_id, to: :mvi, allow_nil: true
 
+    alias :dslogon_edipi
```

Comment:
I think you need to alias this to edipi or else it will always be niil

Reply:
We should also add a test to make sure they are equal

Reply:
The alias is still required because the MVI model calls it when it's doing the Mvi.for_user(self)